Social.API/Controllers/ConversationController.cs

```diff
@@ -163,5 +163,29 @@ public async Task<IActionResult> UpdateConversation(int id, Conversation convers
             }
         }
         
+        [HttpDelete("{id}", Name ="DeleteConversation")]
+        public async Task<IActionResult> DeleteConversationById(int id)
+        {
+        try
+        {
+
+            var conversation = await _repo.GetConversationById(id);
+
+            if(conversation == null){
+
+                return BadRequest($"Could not delete Conversation. Conversation with Id{id} was not found");
```

Comment:
You should probably use NotFound instead of BadRequest. Otherwise it looks good :)